src/index.html

```diff
@@ -943,13 +944,29 @@ <h2>calcite-checkbox</h2>
 
       <calcite-tab>
         <h3>Normal Lockup</h3>
+        <h5>calcite button inside form</h5>
+        <form name="form1" id="form1">
+          <input type="text" name="name1" required />
+          <calcite-button id="submit-button1">Go</calcite-button>
+        </form>
+        <h5>multiple calcite buttons outside form</h5>
+        <form name="form2" id="form2">
+          <input type="text" name="name2" required />
+        </form>
+        <form name="form3" id="form3">
+          <input type="text" name="name3" required />
+        </form>
+        <calcite-button form="form2" id="submit-button2">Go</calcite-button>
```

Comment:
Usually for buttons to submit a form they need to be `type=submit` and inside the form right? 

Why not just give the `calcite-button` a type which could be `submit`, `reset` or `button` by default. Then when placed inside the form they would then submit the form if the type is `submit` and the element is not inline or using a href. No click handler or proxy is necessary then.

Should the inline/href ones be submitting forms anyway?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Reply:
button's default type is submit, IIRC

Reply:
Yeah, buttons are by default `type="submit"`, you can already pass `type="reset"` or `type=button"` to `<calcite-button>` to render it on the child. 

The issue is that the rendered child is in the shadow DOM of `<calcite-button>` and isn't associated with the containing form. The proxy is needed to re-create a button in the light DOM.